app/assets/stylesheets/application.tailwind.css

```diff
@@ -68,4 +66,22 @@
   .typography-micro-m-lh150 {
     @apply text-micro leading-150 font-medium;
   }
+  .btn-primary{
```

Comment:
No `text-white` acho que da para utilizar o branco da nossa palheta: `text-light-palette-p5`, o resultado final não muda em nada hehe

Reply:
o py px fico em dúvida se colocamos aqui mas podemos deixar e no futuro se precisar mudamos

Reply:
Tbm fico com essa dúvida, pois existem botões com espaçamentos internos diferentes. Na dúvida, deixei tudo em uma classe, mas podemos separar, caso for melhor